WIP: Add community portal (#281)
mautic/mautic-community-handbook: docs/mautic_community_portal/mautic_community_portal.rst

> Mautic community portal

Reviewer: ```suggestion
Mautic Community Portal
```

#######################

> The community portal exists at :xref:`community-portal`.

Reviewer: ```suggestion
The Community Portal exists at :xref:`community-portal`.
```

> It's the central place where everything relating to the community happens. Everything from proposing and organizing meetup groups, teams, and working groups to voting on the next members of the Mautic council happens on the portal.

Reviewer: ```suggestion
It's the central place where everything relating to the community happens. Everything from proposing and organizing meetup groups, teams, and working groups to voting on the next members of the Mautic Council happens on the portal.
```

.. vale off

> It's integrated with a single sign-on service, so you can use the same username and password as you use on the forums.

Reviewer: ```suggestion
It's integrated with a single sign-on service, so you can use the same username and password as you use on the Forums.
```

> Members of Mautic have extra privileges such as being able to propose in elections and vote. :xref:`open-collective`, and your email will be synchronized with the system so that you can claim your membership via your user profile.

Reviewer: ```suggestion
Members of Mautic have extra privileges such as being able to stand in elections and vote. :xref:`open-collective`, and you're added to the membership list so that you can claim your membership via your user profile. Note that this is a manual process, so it might take a little time.
```
Active voice, also please remove references to open collective here, because all membership signups should be via https://mau.tc/membership.

> This section contains various tutorials and video resources helping you find your way around the community portal. If something is missing or you are stuck with something please do contact us in #community on :xref:`slack`.

Reviewer: ```suggestion
This section contains various tutorials and video resources helping you find your way around the Community Portal. If something is missing or you are stuck with something please do contact us in #community on :xref:`slack`.
```
